libs/sdk-ui-dashboard/src/model/events/drill.ts

```diff
@@ -50,14 +50,65 @@ export interface DashboardDrillTriggered extends IDashboardEvent {
 /**
  * @alpha
  */
-export function drillTriggered(
+export function drillRequested(
     ctx: DashboardContext,
     drillEvent: IDashboardDrillEvent,
     drillContext: DashboardDrillContext,
     correlationId?: string,
-): DashboardDrillTriggered {
+): DashboardDrillRequested {
     return {
-        type: "GDC.DASH/EVT.DRILL.TRIGGERED",
+        type: "GDC.DASH/EVT.DRILL.REQUESTED",
+        ctx,
+        correlationId,
+        payload: {
+            drillEvent,
+            drillContext,
+        },
+    };
+}
+
+/**
+ * A general drill event that is emitted each time any dashboard drill is requested.
```

Comment:
The comment talks about event being requested, but it is called resolved. Is this intentional?

Reply:
Good catch, I'll correct it.